source/gloperate-glkernel/source/TransparencyKernelStage.cpp

```diff
@@ -0,0 +1,85 @@
+
+#include <gloperate-glkernel/TransparencyKernelStage.h>
+
+#include <algorithm>
+
+#include <glbinding/gl/enum.h>
+
+#include <globjects/Texture.h>
+
+
+namespace
+{
+    const size_t s_alphaValues = 256;
```

Comment:
Those kernel sizes are configurable for the other stages.
Is there a particular reason the kernel sizes are constants here?